opensoundscape/localization.py

```diff
@@ -109,82 +99,61 @@ def estimate_location(
 
         Args:
             algorithm: 'gillette' or 'soundfinder', see localization.localize()
-            cc_threshold: see SpatialEvent documentation
+            cc_threshold: see SpatialEvent documentation. If None, uses self.cc_threshold value
+            cc_filter: filter for generalized cross correlation, see
+                opensoundscape.signal_processing.gcc()
```

Comment:
document what happens for "default" argument (maybe it should be None instead, the typical default value in Python for not specifying an argument)

Reply:
Good catch, the documenation is wrong here.